output/json.go

```diff
@@ -0,0 +1,74 @@
+package output
+
+import (
+	"encoding/json"
+	"github.com/yarpc/crossdock/execute"
+	"github.com/yarpc/crossdock/plan"
+	"io/ioutil"
+)
+
+var StatusNames = map[execute.Status]string{
```

Comment:
nit: can we implement a `String` method on the `execute.Status` type instead?

If we also implement `MarshalText`, then the same type can be used in the `JSONTestReport`